### PR TITLE
Add pmw recipe

### PR DESCRIPTION
**Pull request by mcs07**

### CODE REVIEW

**Comment by mcs07** [comment]
Is it possible to re-trigger appveyor for this recipe? PR #3166 also seems to have stalled.

**Comment by mcs07** [comment]
This is ready for review

File: recipes/pmw/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - Pmw
+
+about:
+  home: http://pmw.sourceforge.net/
+  license: BSD
```
Comment (ocefpaf) [inline recipes/pmw/meta.yaml]: Do you know the license version? `BSD 2-Clause` or `BSD 3-Clause`?

Comment (mcs07) [inline recipes/pmw/meta.yaml]: @ocefpaf It is tagged as BSD on PyPI, so I used that. But I just looked in the actual package, and the license file (now specified) contains what appears to be the MIT license, so I've changed the license fields to that.

Comment (ocefpaf) [inline recipes/pmw/meta.yaml]: > It is tagged as BSD on PyPI, so I used that.

`PyPI` is, IMO, broken when it comes to the BSD license b/c it does not require the version for `BSD`.

> But I just looked in the actual package, and the license file (now specified) contains what appears to be the MIT license, so I've changed the license fields to that.

That is bad :unamused:

Can you report that upstream?

Comment (mcs07) [inline recipes/pmw/meta.yaml]: Posted to their mailing list, but the project doesn't seem to have got a lot of attention in the last few years...

Comment (ocefpaf) [inline recipes/pmw/meta.yaml]: No problem. There is so much we can do on the packaging side.

This is good to go. I will merge as soon as the CIs are green.

Thanks!

**Pull request merged**